### PR TITLE
Closes #53: Created lab test result files with alpha synuclein results

### LINKED ISSUE #53: Feature Request:  Add CSF alpha synuclein pathology
### Feature Idea

Amprion CSF seed amplification assay [SAA](https://www.ampriondx.com/) recently approved by the FDA to measure alpha-synuclein pathology in Parkinson disease, Lewy body dementia, and Alzheimer's disease. 

### Relevant Input

SAA reports binary positive or negative and it has variable prevalence in amyloid positive individuals. For our nv_neuro dataset, we can set reasonable set 30% as SAA positive in amyloid positive records, and 10% as SAA negative in amyloid negative records. 

### Relevant Output

_No response_

### Reproducible Example/Pseudo Code

_No response_

### PR DESCRIPTION
**Pull request by jackychen651**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/CONTRIBUTING.html#detailed-development-process) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral family codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `main` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` under the header `# admiral<ext> (development version)` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the Reference page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

File: data-raw/lb_neuro.R [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+  dplyr::filter(VISIT == "BASELINE") %>%
+  dplyr::select(USUBJID, VISITNUM, VISIT, NVDTC, NVDY) %>%
+  dplyr::group_by(USUBJID, VISIT) %>%
+  dplyr::arrange(as.Date(NVDTC)) %>%
+  dplyr::slice(1) %>%
+  dplyr::ungroup() %>%
+  dplyr::rename(LBDTC = NVDTC, LBDY = NVDY) %>%
+  distinct()
+
+# Derive treatment start date to calculate VISITDY (following adsl_neuro.R pattern)
```
Comment (jwang-lilly) [inline data-raw/lb_neuro.R]: How does the treatment information matter for baseline alpha-synuclein data? I didn't see treatment info in lb_neuro.

File: data-raw/lb_neuro.R [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+    SAAPOSDT = suppressWarnings(min(as.Date(LBDTC[LBSTRESC == "Positive"]), na.rm = TRUE)),
+    .groups = "drop"
+  ) %>%
+  dplyr::mutate(
+    SAAPOSDT = ifelse(is.infinite(SAAPOSDT), NA, SAAPOSDT),
+    SAAPOSDT = as.character(SAAPOSDT)
+  )
+
+# Validation checks
+expected_visits <- c("BASELINE", "WEEK 12", "WEEK 26")
```
Comment (jwang-lilly) [inline data-raw/lb_neuro.R]: why are WEEK 12 WEEK 26 if only baseline data is created?

File: data-raw/lb_neuro.R [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+  dplyr::select(USUBJID, VISITNUM, VISIT, NVDTC, NVDY) %>%
+  dplyr::group_by(USUBJID, VISIT) %>%
+  dplyr::arrange(as.Date(NVDTC)) %>%
+  dplyr::slice(1) %>%
+  dplyr::ungroup() %>%
+  dplyr::rename(LBDTC = NVDTC, LBDY = NVDY) %>%
+  distinct()
+
+# Derive treatment start date to calculate VISITDY (following adsl_neuro.R pattern)
+ex_ext <- ex %>%
```
Comment (jwang-lilly) [inline data-raw/lb_neuro.R]: why treatment dates matter

**Comment by jwang-lilly** [comment]
@jackychen651 Excellent work! I added a few comments with some thoughts on if we might simplify some parts of the implementation.

You may want to keep an eye on the pull request “Close #87 Fixed codes after the new lint check #99.” If it’s merged, the same update would likely apply to lb_neuro.R as well.

**Comment by jackychen651** [comment]
@jwang-lilly Thanks for your suggestions. I will review them later today or tomorrow!

File: data-raw/lb_neuro.R (hunk 1 above)
Comment (jackychen651) [reply data-raw/lb_neuro.R]: You are right. There is no point using treatment date information.`VISITDY` can be simply inherited from `NVDY` and then filter out the observational data.

File: data-raw/lb_neuro.R (hunk 2 above)
Comment (jackychen651) [reply data-raw/lb_neuro.R]: Yes we can safely deleted that.

**Comment by jwang-lilly** [comment]
@jackychen651, team decides this branch will not be merged to admiralneuro main. Instead, pharmaversesdtm will be updated with alpha-synuclein data. After you are added to pharmaversesdtm, you will be assigned to issue #203 [add admiralneuro dataset to include AMPRION alpha-synuclein test data](https://github.com/pharmaverse/pharmaversesdtm/issues/203)
